waku-bindings/tests/node.rs

```diff
@@ -29,8 +29,14 @@ pub fn main() -> Result<(), String> {
         min_peers_to_publish: None,
         filter: None,
         log_level: Some(WakuLogLevel::Error),
+        discv5: Some(false),
```

Comment:
@al8n , I think it fails because it is not activatedl. Maybe this should be `Some(true)`?

Reply:
Yeah! DiscV5 needs to be enabled for Start() / Stop() to work!

Reply:
Looking at `go-waku` it seems that discoveryV5 is mounted during the `start` of a node. In this case the node is only initialized, but not started, thus `discoveryV5` is not yet mounted, even if the config has it enabled.
It should look something like this:
```
let node = waku_new(Some(config))?;
let node = node.start()?;
println!("Node peer id: {}", node.peer_id()?);

let result = node.discv5_start()?;
println!("Discv5 started: {}", result);
```
But in this case `discv5_start()` method should be implemented for `WakuNodeHandle<Running>` variant. 
I'm not sure if that's how things should look.